ts-tests/scripts/clean-up.sh

```diff
@@ -1,19 +1,20 @@
 #!/bin/sh
 
-TMP_DIR="$1"
+TMP_DIR="/tmp"
+TMP_DIR="${1:-$TMP_DIR}"
```

Comment:
could be combined to 
```suggestion
TMP_DIR="${1:-/tmp}"
```